pkg/rancher-desktop/backend/kube/client.ts

```diff
@@ -7,14 +7,11 @@ import util from 'util';
 
 import * as k8s from '@kubernetes/client-node';
 
-import Logging from '../utils/logging';
+import Logging from '@pkg/utils/logging';
```

Comment:
This looks to be part of a separate commit to use `typeUtils/defined` instead of our own `defined` here.

Reply:
Extracted to 7c5ceb19d495a5f031ef7a6340a05d5dea68779a.